packages/mdc-slider/foundation.js

```diff
@@ -177,7 +177,7 @@ class MDCSliderFoundation extends MDCFoundation {
       if (indivisible) {
         const lastStepRatio = (max - numMarkers * step) / step + 1;
         const flex = getCorrectPropertyName(window, 'flex');
-        this.adapter_.setLastTrackMarkersStyleProperty(flex, lastStepRatio);
+        this.adapter_.setLastTrackMarkersStyleProperty(flex, String(lastStepRatio));
```

Comment:
It might make more sense to do the `String()` conversion _inside of_ `setLastTrackMarkersStyleProperty()` (right before it gets passed to `setProperty()`). That way, callers can pass a `string` _or_ a `number`, and `setLastTrackMarkersStyleProperty()` will deal with the conversion to make Closure Compiler happy.

Reply:
That would require making an adapter API smarter, and we want our adapter APIs to be as simple and easy to implement as possible.

Reply:
In either case, Closure Compiler was happy. Even when run with the internal blaze compiler, it was fine with both the coerced string and the `string|number`.